diff --git a/sysdeps/ia64/fpu/e_exp.S b/sysdeps/ia64/fpu/e_exp.S
index c106a2dcde..b2d8f7de49 100644
--- a/sysdeps/ia64/fpu/e_exp.S
+++ b/sysdeps/ia64/fpu/e_exp.S
@@ -737,6 +737,7 @@ EXP_UNDERFLOW_ZERO:
 ;;
 
 GLOBAL_IEEE754_END(exp)
+libm_hidden_def (__exp)
 libm_alias_double_other (__exp, exp)
 #ifdef SHARED
 .symver exp,exp@@GLIBC_2.29
diff --git a/sysdeps/ia64/fpu/e_expf.S b/sysdeps/ia64/fpu/e_expf.S
index a6894c3280..49f0f5fa5f 100644
--- a/sysdeps/ia64/fpu/e_expf.S
+++ b/sysdeps/ia64/fpu/e_expf.S
@@ -656,6 +656,7 @@ EXP_UNDERFLOW_ZERO:
 ;;
 
 GLOBAL_IEEE754_END(expf)
+libm_hidden_def (__expf)
 libm_alias_float_other (__exp, exp)
 #ifdef SHARED
 .symver expf,expf@@GLIBC_2.27
diff --git a/sysdeps/ia64/fpu/s_expm1l.S b/sysdeps/ia64/fpu/s_expm1l.S
index 769d0ff15e..7d954607aa 100644
--- a/sysdeps/ia64/fpu/s_expm1l.S
+++ b/sysdeps/ia64/fpu/s_expm1l.S
@@ -1378,6 +1378,7 @@ EXP_64_UNSUPPORTED:
 ;;
 
 GLOBAL_IEEE754_END(expl)
+libm_hidden_def (__expl)
 libm_alias_ldouble_other (__exp, exp)
 
 LOCAL_LIBM_ENTRY(__libm_error_region)
